src/library-authoring/data/api.ts

```diff
@@ -418,8 +418,8 @@ export async function createCollection(libraryId: string, collectionData: Create
  * Fetch the OLX for the given XBlock.
  */
 // istanbul ignore next
-export async function getXBlockOLX(usageKey: string): Promise<string> {
-  const { data } = await getAuthenticatedHttpClient().get(getXBlockOLXApiUrl(usageKey));
+export async function getXBlockOLX(usageKey: string, version?: string): Promise<string> {
```

Comment:
Instead of `version?: string` maybe we can use `version?: VersionSpec` ?

https://github.com/openedx/frontend-app-authoring/blob/f86c609ff147348e15c1a51bbd64f79b4693bdcc/src/library-authoring/LibraryBlock/LibraryBlock.tsx#L7

Reply:
Updated https://github.com/openedx/frontend-app-authoring/pull/1534/commits/73ed8461ee4ad76faee9f1e106f156ac00ef3bd9